### ISSUE TITLE
Manifest.json for web app installation

### ISSUE THREAD

**Issue by lucianmarin** [lobsters/lobsters#1287]
Can you a manifest.json to be able to install it on Android/iOS?

```html
<link rel="manifest" type="application/json" href="/static/manifest.json">
```

Below is the code I use on [Subreply](https://subreply.com/) and it works great. Maskable icon is the same as the favicon one, but zoomed out by 1.333x. Everything else is pretty obvious and straightforward.

```json
{
  "lang": "en",
  "name": "Subreply",
  "short_name": "Subreply",
  "description": "Tiny, but mighty social network.",
  "icons": [
    {
      "src": "/static/192.png",
      "sizes": "192x192",
      "type": "image/png",
      "purpose": "any"
    },
    {
      "src": "/static/256.png",
      "sizes": "256x256",
      "type": "image/png",
      "purpose": "maskable"
    }
  ],
  "id": "/feed",
  "start_url": "/feed",
  "display": "standalone",
  "orientation": "portrait",
  "status": "ok"
}
```

**Comment by pushcx** [comment]
If it's that easy, sounds like a nice convenience to add. Would you like to open a PR?

Anything to be aware of for updates? I remember seeing a headline or two go by about bookmarking sites but I don't remember particulars.

**Comment by lucianmarin** [comment]
I did, but I have no idea how to link PR to this issue.

**Comment by pushcx** [comment]
Mention the idea of one from the other in the text of comment or commit message with the `#`, like:

#1288

**Comment by kaycebasques** [comment]
followup from https://lobste.rs/s/k2tyg5/lobsters_pwa#c_zkksaw

![Screenshot_20240718-080647](https://github.com/user-attachments/assets/3171e484-c720-4d72-a081-c207e7a02aca)

**Comment by pushcx** [comment]
I removed the `manifest.json` in e121f808, users [reported](https://lobste.rs/s/k2tyg5) intrusive popups on Android; more info in the site story.

**Comment by mrexodia** [comment]
Would you be open to reconsider adding back PWA support? It makes the iOS experience much better. I’d be happy to put in some work also checking if Android still has popups. From what I remember Google kinda steered away from PWAs, so it might have improved since.

**Comment by pushcx** [comment]
Yeah, I am still happy to offer the convenience if it's not going to cause a confusing UI or a maintenance burden.

**Comment by pushcx** [comment]
This was reverted and closed in 2024 ago when we found Android had a very strange UI, but it's worth taking another look at.

If you'd prefer not to be alerted to activity, github has an 'unsubscribe' button in the right column

**Comment by pushcx** [comment]
@lucianmarin Please use your words.

**Comment by pushcx** [comment]
I didn't recognize the handle, but he was banned a while back for spamming the site, I guess that's what the closing was about. Welp, reported and banned from here, too. GitHub doesn't allow me to clean up his distracting abuse from the issue timeline.
